db/seeds.rb

```diff
@@ -12,7 +12,7 @@
 
 puts "Creating Users"
 ["goh", "kyla", "yaamani", "jielin"].each do |user|
-  User.create(username: user, email: "#{user}@playsnap.com", password: 123456)
+  User.create(username: user, email: "#{user}@playsnap.com", password: "playsnap")
```

Comment:
<img width="452" alt="password" src="https://user-images.githubusercontent.com/99309242/189511054-c10ea3a7-d67f-4dcb-a523-f125c6dfcebd.png">

Reply:
Change pw because this password `123456` is not safe to use and not look good during demo